test/WebJobs.Extensions.Tests/Extensions/Timers/Listener/TimerListenerTests.cs

```diff
@@ -270,7 +270,23 @@ public void Timer_VerifyMaxInterval()
             Assert.True(updateCalled);
         }
 
-        private void CreateTestListener(string expression, bool useMonitor = true)
+        [Fact]
+        public async Task StoppedListener_DoesNotContinueRunning()
```

Comment:
I ran it ~15 times and it failed every time.

---
In reply to: [78817781](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/149#discussion_r78817781) [](ancestors = 78817781)